Update EIP-5298: Move to Review (#6343)
ethereum/EIPs: EIPS/eip-5298.md

}
```

3. `claimTo` MUST check if `msg.sender` is the owner of the ENS node identified by `bytes32 ensNode` (and/or approved by the domain in implementation-specific ways). The compliant contract then MUST make a call to the `safeTransferFrom` function of [EIP-721](./eip-712.md) or [EIP-1155](./eip-1155.md).

4. Any `ensNode` is allowed.

## Rationale

1. ENS was chosen because it is a well-established scoped ownership namespace.
This is nonetheless compatible with other scoped ownership namespaces.

> 2. We didn't expose getters or setters to an ensRoot because we believe that shall be address outside of the scope of this EIP.

Reviewer: ```suggestion
2. We didn't expose getters or setters for ensRoot because it is outside of the scope of this EIP.
```

## Security Considerations

> Needs disucssion.

Reviewer: ```suggestion
Needs discussion.
```
